docs/get-started/installation.md

```diff
@@ -150,6 +150,46 @@ docker run --rm quickwit/quickwit --version
 # You can also safely ignore jemalloc warnings.
 docker run --rm --platform linux/amd64 quickwit/quickwit --version
 ```
+Here is the directory layout of Quickwit inside the container in case you need to mount a volume:
+
+```bash
+/quickwit
+    ├── config
+    │   └── quickwit.yaml
+    ├── quickwit
+    └── qwdata
+```
+
+- `config/quickwit.yaml`: the default configuration file.
+- `quickwit`: the quickwit executable binary.
+- `qwdata/`: the default data directory.
+
+Here is a list of the available environment variables and ports options that can be used to configure Quickwit:
+
+[Node configuration](../configuration/node-config.md):
+- `QW_CLUSTER_ID`
+- `QW_NODE_ID`
+- `QW_ENABLED_SERVICES`
+- `QW_LISTEN_ADDRESS`
+- `QW_ADVERTISE_ADDRESS`
+- `QW_PEER_SEEDS`
+- `QW_DATA_DIR`
+- `QW_METASTORE_URI`
+- `QW_DEFAULT_INDEX_ROOT_URI`
+- `QW_ENABLE_JAEGER_EXPORTER`
+- `QW_ENABLE_JAEGER_ENDPOINT`
+- `QW_S3_ENDPOINT`
+- `QW_S3_MAX_CONCURRENCY`
+- `QW_AZURE_STORAGE_ACCOUNT`
+- `QW_AZURE_STORAGE_ACCESS_KEY`
+
+[Ports configuration](../configuration/ports-config.md):
+- `QW_REST_LISTEN_PORT`
```

Comment:
Same for ports configuration, it's nice to have the link but I definitely want to avoid env variables in this page.